### PR TITLE
CON-2299-2300-2302-2729 epao withdrawal admin tabs

### PR DESCRIPTION
**Pull request by ben1stone**

### CODE REVIEW

File: src/SFA.DAS.AdminService.Web/Views/Application/WithdrawalApplication/WithdrawalApplications.cshtml [hunk 1]
```diff
@@ -5,11 +5,11 @@
 @model SFA.DAS.AdminService.Web.ViewModels.Apply.Applications.ApplicationsDashboardViewModel
 
 @{
-    ViewBag.Title = "Organisation withdrawal applications";
+    ViewBag.Title = "Withdrawal applications";
     Layout = "_Layout";
 }
 
-<a asp-action="Index" asp-controller="Dashboard" class="govuk-back-link">Return to dashboard</a>
+<a asp-action="Index" asp-controller="Dashboard" class="govuk-back-link">Back</a>
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AdminService.Web/Views/Application/WithdrawalApplication/WithdrawalApplications.cshtml:12]: There is an AC for this which does not mention it changing on each story

File: src/SFA.DAS.AdminService.Web/Views/Application/WithdrawalApplication/_WithdrawalApplicationsPartial.cshtml [hunk 2]
```diff
@@ -21,28 +21,26 @@
...
                                             sfa-sort-column="@WithdrawalApplicationsSortColumn.Name"
                                             sfa-table-sort-column="@Model.SortColumn"
-                                            sfa-sort-direction="@Model.SortDirection">Name</sfa-sortable-column-header>
+                                            sfa-sort-direction="@Model.SortDirection">Organisation</sfa-sortable-column-header>
             </th>
-            <th scope="col" class="govuk-table__header govuk-!-width-one-quarter govuk-table__header--numeric">
+            <th scope="col" class="govuk-table__header govuk-!-width-two-thirds">
                 <sfa-sortable-column-header class=@(Model.Fragment + "-sort govuk-link" + " das-no-wrap")
-                                            data-sortColumn="@WithdrawalApplicationsSortColumn.GovernanceRecommendation"
+                                            data-sortColumn="@WithdrawalApplicationsSortColumn.Type"
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AdminService.Web/Views/Application/WithdrawalApplication/_WithdrawalApplicationsPartial.cshtml:35]: Why is this not the name of the column - 'Details'

File: src/SFA.DAS.AdminService.Web/Views/Application/WithdrawalApplication/_WithdrawalApplicationsPartial.cshtml [hunk 3]
```diff
@@ -65,50 +63,44 @@
...
+                        </a>
+                    </td>
+                    <td class="govuk-table__cell">
+                        @if (application.SequenceNo == ApplyConst.ORGANISATION_WITHDRAWAL_SEQUENCE_NO)
+                        {
+                            <text>Withdrawal from register</text>
+                        }
+                        else if (application.SequenceNo == ApplyConst.STANDARD_WITHDRAWAL_SEQUENCE_NO)
+                        {
+                            @($"{application.StandardName} (ST{application.StandardCode})")
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AdminService.Web/Views/Application/WithdrawalApplication/_WithdrawalApplicationsPartial.cshtml:93]: Should this StandardCode - the code is different from the Reference?

File: src/SFA.DAS.AdminService.Web/Views/Application/OrganisationApplication/OrganisationApplications.cshtml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 @using SFA.DAS.AdminService.Common.Extensions;
-@using SFA.DAS.AdminService.Web.Helpers;
+@using SFA.DAS.AdminService.Web.Helpers;organisation applications
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AdminService.Web/Views/Application/OrganisationApplication/OrganisationApplications.cshtml:2]: What's this?

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.AdminService.Web/Views/Application/WithdrawalApplication/WithdrawalApplications.cshtml (hunk 1 above)
Comment (ben1stone) [reply src/SFA.DAS.AdminService.Web/Views/Application/WithdrawalApplication/WithdrawalApplications.cshtml]: It was changed to back in the content deck but I've confirmed it should be 'Return to dashboard' so I've reverted

File: src/SFA.DAS.AdminService.Web/Views/Application/OrganisationApplication/OrganisationApplications.cshtml (hunk 4 above)
Comment (ben1stone) [reply src/SFA.DAS.AdminService.Web/Views/Application/OrganisationApplication/OrganisationApplications.cshtml]: Removed

File: src/SFA.DAS.AdminService.Web/Views/Application/WithdrawalApplication/_WithdrawalApplicationsPartial.cshtml (hunk 3 above)
Comment (ben1stone) [reply src/SFA.DAS.AdminService.Web/Views/Application/WithdrawalApplication/_WithdrawalApplicationsPartial.cshtml]: I've replaced with Standard reference. Std ref was already returned from the API as part of the ApplicationSummaryItem

File: src/SFA.DAS.AdminService.Web/Views/Application/WithdrawalApplication/_WithdrawalApplicationsPartial.cshtml (hunk 2 above)
Comment (ben1stone) [reply src/SFA.DAS.AdminService.Web/Views/Application/WithdrawalApplication/_WithdrawalApplicationsPartial.cshtml]: I've renamed the column sort type

**Review by ChrisJWoodcock — APPROVED** [review]

**Pull request merged**